helm_deploy/hmpps-interventions-ui/templates/_envs.tpl

```diff
@@ -7,23 +7,35 @@ env:
   - name: INGRESS_URL
     value: "https://{{ .Values.ingress.host }}/"
 
-  - name: CLIENT_ID
+  - name: API_CLIENT_ID
     valueFrom:
       secretKeyRef:
         name: hmpps-auth
-        key: client-id
+        key: interventions-ui-client-id
 
-  - name: CLIENT_SECRET
+  - name: API_CLIENT_SECRET
     valueFrom:
       secretKeyRef:
         name: hmpps-auth
-        key: client-secret
+        key: interventions-ui-client-secret
+
+  - name: LOGIN_CLIENT_ID
+    valueFrom:
+      secretKeyRef:
+        name: hmpps-auth
+        key: interventions-login-client-id
+
+  - name: LOGIN_CLIENT_SECRET
+    valueFrom:
+      secretKeyRef:
+        name: hmpps-auth
+        key: interventions-login-client-secret
 
   - name: SESSION_SECRET
     valueFrom:
       secretKeyRef:
         name: session
-        key: ui-session-secret
+        key: interventions-ui-session-secret
```

Comment:
are these secret names better for you @sldblog ?

Reply:
Really sorry @tomsmyers, my mistake, I wasn't clear before

I meant the `name` instead of the `key`, from the use case of finding out which secret is used by which service. If we intend to share `session` between UI and service then all is well.

```
$ k get secrets --show-labels=true --namespace=hmpps-interventions-dev
NAME                                                TYPE                                  DATA   AGE     LABELS
circleci-token-24d6f                                kubernetes.io/service-account-token   3      9d      <none>
default-token-gk759                                 kubernetes.io/service-account-token   3      12d     <none>
hmpps-auth                                          Opaque                                6      83m     <none>
session                                             Opaque                                1      82m     <none>
...[snip helm stuff]...
```

`key: ui-session-secret` is a great name -- sorry again, I wasn't looking at `key` before.